aries-site/src/examples/templates/wizard/WizardValidationExample.js

```diff
@@ -0,0 +1,391 @@
+import React, { useContext, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  CheckBoxGroup,
+  Form,
+  FormField,
+  Header,
+  Heading,
+  Layer,
+  List,
+  RadioButtonGroup,
+  ResponsiveContext,
+  Select,
+  Text,
+  TextArea,
+  TextInput,
+} from 'grommet';
+import {
+  Checkmark,
+  CircleAlert,
+  FormClose,
+  FormNextLink,
+  FormPreviousLink,
+} from 'grommet-icons';
+
+const defaultFormValues = {
+  'text-input-validation': '',
+  'radio-button-group-validation': '',
+  select: '',
+  checkboxgroup: '',
+  'text-area': '',
+};
+
+const stepOneValidate = values => {
+  const emailRegex = RegExp(
+    '[^@ \\t\\r\\n]+@[^@ \\t\\r\\n]+\\.[^@ \\t\\r\\n]+',
+  );
+  const emailValid = emailRegex.test(values['text-input-validation']);
+
+  return {
+    email: emailValid ? '' : 'Invalid email address.',
+    isValid: !!emailValid,
+  };
+};
+
+const StepOne = ({ activeIndex, setActiveIndex, formValues }) => {
+  const [error, setError] = useState({
+    email: '',
+    radiobuttongroup: '',
+    isValid: true,
+  });
+
+  return (
+    <>
+      <Box margin={{ bottom: 'large' }}>
```

Comment:
Does the gap between the field and error seems too large?
![Screen Shot 2020-08-06 at 11 02 40 AM](https://user-images.githubusercontent.com/1756948/89560648-99b47300-d7d4-11ea-84b8-0713543188d7.png)

If so, a few potential solutions:
- Move the error to appear within the Box
- Make this `bottom: 'medium'` and give the Button a `margin={{top: 'medium'}}`
- probably some others

Reply:
I would agree. I've tightened the spacing to better match the example here: https://www.figma.com/file/3fkwBelW5UsCbfwdDCJkT8/HPE-Form-Templates?node-id=2668%3A3547